wordpress-mobile/WordPress-Android
# Issue/13329 expose backup feature

Parent #13329 

This PR adds exposes a temporary entry point for development purposes to the Backup Download feature. When the `BackupFeatureConfig` is enabled, the entry point can be accessed via the Activity Log action item click. The following dialog will be launched and the user can navigate to Backup download or continue using the rewind site feature.

<img src="https://user-images.githubusercontent.com/506707/101049215-51724b00-3551-11eb-8ad6-e4129da8f578.png" width="200" alt="Screenshot">

In a future iteration the dialog box will be removed and a true More menu will be displayed. There is a line item task for this in the parent issue.

**To test:**
- Launch the App
- Navigate to Activity Log
- Notice that the action button icon still shows the rewind icon
- Head over to App Settings -> test feature configuration
- Enable the `BackupFeatureConfig` and restart the app
- Navigate back to Activity Log
- Notice that the action button is now the 3 dot vertical more icon
- Tap on the icon to show the dialog box
- Tap the "Create downloadable backup" option (You can still select the normal rewind action from the dialog)
- The backup detail view is shown (The feature itself is not functional, this is just to test that the view shows)

PR submission checklist:
